Dockerfile

```
@@ -23,20 +23,21 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   && echo "🔍 Checking FFmpeg license (builder stage)..." \
   && ffmpeg -version | grep -E "enable-gpl|enable-nonfree" && (echo "❌ GPL/nonfree FFmpeg detected!" && exit 1) || echo "✅ LGPL FFmpeg build verified."
 
-# Copy requirements and install Python deps
-COPY requirements/ requirements/
-RUN pip install --no-cache-dir --upgrade pip \
-  && pip install --no-cache-dir -r requirements/prod.txt
+# Use system Python so venv symlinks point to /usr/local/bin (survives COPY to runtime)
+COPY uv.lock pyproject.toml .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)